master/getting-started/kubernetes/installation/other.md

```diff
@@ -31,35 +31,31 @@ complete the following steps.
 
 1. If your cluster contains more than 50 nodes:
 
-   - In the `ConfigMap` named `calico-config`, locate the `typha_service_name`,
-     delete the `none` value, and replace it with `calico-typha`.
-
-   - Modify the replica count in the`Deployment` named `calico-typha`
-     to the desired number of replicas.
-
-     ```
-     apiVersion: apps/v1beta1
-     kind: Deployment
-     metadata:
-       name: calico-typha
-       ...
-     spec:
-       ...
-       replicas: <number of replicas>
-     ```
-     {: .no-select-button}
-
-     We recommend at least one replica for every 200 nodes and no more than
-     20 replicas. In production, we recommend a minimum of three replicas to reduce
-     the impact of rolling upgrades and failures.  The number of replicas should
-     always be less than the number of nodes, otherwise rolling upgrades will stall.
-     In addition, Typha only helps with scale if there are fewer Typha instances than
-     there are nodes.
-
-     > **Tip**: If you set `typha_service_name` without increasing the replica
-     > count from its default of `0` Felix will try to connect to Typha, find no
-     > Typha instances to connect to, and fail to start.
-     {: .alert .alert-success}
+   By default the replica count in the `Deployment` named `calico-typha` is set to 1. 
+   You may want to consider changing this for large clusters or production environments.
+   
+   ```
+   apiVersion: apps/v1beta1
```

Comment:
Can you make this `apps/v1`? `apps/v1beta` is deprecated and will be removed from the next k8s version.